src/helpers/sorts.ts

```diff
@@ -103,6 +103,16 @@ export function getFirstMeasureSort(afm: AFM.IAfm): AFM.SortItem[] {
     return [];
 }
 
+export function getDefaultAttributeGeoPushpinSort(afm: AFM.IAfm): AFM.SortItem[] {
+    const attributes: AFM.IAttribute[] = get(afm, "attributes");
+
+    if (attributes.length === 2 || attributes.length === 3) {
+        return getAttributeSortItems(attributes[1].localIdentifier, ASC);
```

Comment:
what if `attributes[1]` is tooltipText, not segmentBy?

Reply:
we have checked condition at [here](https://github.com/gooddata/gooddata-react-components/blob/d5f702262048b60f0aa63b94fa4530af63e4cd86/src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx#L269), so attributes always have segmentBy

Reply:
Who know if in the future, somebody will call this function without checking that condition :)
Please move that condition into this function